packages/docs/getting-started/running-a-validator-baklava.md

```diff
@@ -0,0 +1,178 @@
+
+This section is specific for Baklava Network. You can find more details about running a Validator in different networks at [Running a Validator page](running-a-validator.md).
+
+## Instructions
+
+If you are re-running these instructions, the Celo Docker image may have been updated, and it's important to get the latest version.
+
+To run a complete Validator it's necessary to execute the following components:
+
+- The Validator software
+- A Proxy that acts as an intermediary for the Validator requests
+- The Attestation Service
+
+The Proxy is not mandatory but highly recommended. It allows to protect the Validator node from outside connections and hide the Validator behind that Proxy from other nodes of the network.
+
+### Environment variables
+
+| Variable                     | Explanation                                                               |
+| ---------------------------- | ------------------------------------------------------------------------- |
+| CELO_IMAGE                   | The Docker image used for the Validator and Proxy containers              |  |
+| NETWORK_ID                   | The Celo network chain ID                                                 |  |
+| URL_VERIFICATION_POOL        | URL for the Verification pool for the attestation process                 |  |
+| CELO_VALIDATOR_GROUP_ADDRESS | The etherbase public address for the validation group                     |  |
+| CELO_VALIDATOR_ADDRESS       | The etherbase public address for the Validator instance                   |  |
+| CELO_PROXY_ADDRESS           | The etherbase public address for the Proxy instance                       |  |
+| CELO_VALIDATOR_POP           |                                                                           |  |
+| PROXY_ENODE                  | The ethereum node address for the Validator                               |  |
+| PROXY_IP                     | The Proxy container internal IP address from docker pool address          |  |
+| ATTESTATION_KEY              | The etherbase private key for the account used in the Attestation Service |  |
+| ATTESTATION_SERVICE_URL      | The URL to access the Attestation Service deployed                        |  |
+| METADATA_URL                 | The URL to access the metadata file for your Attestation Service          |  |
+
+First we are going to setup the main environment variables related with the `Baklava` network. Run:
+
+```bash
+export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:baklava
+export NETWORK_ID=1101
+```
+
+### Pull the Celo Docker image
+
+In all the commands we are going to see the `CELO_IMAGE` variable to refer to the right Docker image to use. Now we can get the Docker image:
+
+```bash
+docker pull $CELO_IMAGE
+```
+
+### Create accounts
```

Comment:
IMO this section should cover the creation of the attestation key

Reply:
To avoid duplicate this documentation in two pages, I added the text clarifying are necessary 4 accounts (Validator, Validator Group, Proxy and Attestation Service) and added a link to the attestation service documentation page.